src/utils/mesh.js

```diff
@@ -146,18 +158,18 @@ function getPaaSMeshConfig (core, catalog, githubOrg, githubRepo) {
                     "endpoint": "${catalog}",
                     "useGETForQueries": true,
                     "schemaHeaders": {
-                      "Content-Type": "application/json",
-                      "x-api-key": "4dfa19c9fe6f4cccade55cc5b3da94f7"
+                        "Content-Type": "application/json",
+                        "x-api-key": "${apiKey}",
                     },
                     "operationHeaders": {
-                      "Content-Type": "application/json",
-                      "Magento-Environment-Id": "{context.headers['magento-environment-id']}",
-                      "Magento-Website-Code": "{context.headers['magento-website-code']}",
-                      "Magento-Store-View-Code": "{context.headers['magento-store-view-code']}",
-                      "Magento-Store-Code": "{context.headers['magento-store-code']}",
-                      "Magento-Customer-Group": "{context.headers['magento-customer-group']}",
-                      "x-api-key": "{context.headers['x-api-key']}",
-                      "Authorization": "context.headers['authorization']"
+                        "Content-Type": "application/json",
+                        "Magento-Environment-Id": "${environmentId}",
+                        "Magento-Website-Code": "{context.headers['magento-website-code']}",
+                        "Magento-Store-View-Code": "{context.headers['magento-store-view-code']}",
+                        "Magento-Store-Code": "{context.headers['magento-store-code']}",
+                        "Magento-Customer-Group": "{context.headers['magento-customer-group']}",
+                        "x-api-key": "${apiKey}",
```

Comment:
I think these should still use the context.headers, for all.

So for example `schemaHeaders.x-api-key = "{context.headers['x-api-key'}"`.
And for `Magento-Environment-Id` we can just leave as is - the value should be what is passed in by the request, not some value we define during the provisioning.

Reply:
There is a small difference between schema and operation headers.

Schema headers are used while fetching and building the schema using the introspection query. Hence these can not be dynamic headers, they need to be defined on the mesh while building. That was the reason why I hard coded an API Key earlier.

Operation headers are used at runtime while calling the source with graphql operations (queries and mutations). Hence these can be dynamic (context.headers) where the value of that header can be defined in the request.

Given this, maybe we can use `config.commerce.apiKey` as the schema header since its needed at build time and continue using `context.headers[API_Key]` for operation headers (runtime headers).

Would that be acceptable?

Reply:
That said what shall I do about the Magento env ID? Should I use the value from `config.commerce` or take it from the request `context.headers` at runtime?

Reply:
We should, as much as possible, not use static values that are defined once. So for the runtime headers, always use context values. For schema headers, I think I have more questions.

1. Are schema headers needed for both PaaS/CS and SaaS meshes?
2. What schema headers are required for either? Where are they defined?
3. How can we ensure that we get these values and customers know what they are giving us these values for? I would prefer we ask specifically that the user of the CLI provide these values and that they know they are used to generate the mesh schema.